packages/in_app_purchase/in_app_purchase_storekit/lib/src/types/app_store_purchase_param.dart

```diff
@@ -32,4 +33,7 @@ class AppStorePurchaseParam extends PurchaseParam {
 
   /// Quantity of the product user requested to buy.
   final int quantity;
+
+  /// Discount applied to the product (optional).
```

Comment:
nits, the "optional" keyword is probably not necessary as the nullability of this parameter already indicates it is optional.
However, it might be better to add an explanation what does it mean to be null, something like:
```
///
/// The value is `null` when the product does not have a discount.
```